README.md

```diff
@@ -1,28 +1,63 @@
 # CodiMD for YunoHost
 
-A collaborative editor to work on notes written in Markdown
+[![Integration level](https://dash.yunohost.org/integration/codimd.svg)](https://dash.yunohost.org/appci/app/codimd) ![](https://ci-apps.yunohost.org/ci/badges/codimd.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/codimd.maintain.svg)  
+[![Install CodiMD with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=codimd)
+
+*[Lire ce readme en français.](./README_fr.md)*
+
+> *This package allows you to install CodiMD quickly and simply on a YunoHost server.  
+If you don't have YunoHost, please consult [the guide](https://yunohost.org/#/install) to learn how to install it.*
+
+## Overview
+CodiMD is a real-time collaborative word processing web service. It uses Markdown language.
 
 **Shipped version:** 1.6.0
 
-**Status**: In progress, do *not* consider this app as stable and fully working (yet)
+## Screenshots
 
-**To be implemented**:
-- Upgrade (right now you need to remove the app and install its new version - your documents are likely to be lost…)
-- Backup and restore system
+![](https://demo.codimd.org/screenshot.png)
 
-Links:
-- [Yunohost project](https://yunohost.org)
-- [CodiMD website](https://github.com/codimd/server/)
-- [Demo](https://demo.codimd.org/)
+## Demo
 
-[![Install CodiMD with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=codimd)
+* [Official demo](https://demo.codimd.org/)
 
+## Configuration
 
-### Screenshot
+You can configure CodiMD by editing this file `/var/www/codimd/config.json` using this the [documentation](https://github.com/codimd/server/blob/master/docs/configuration-config-file.md)
 
-![](https://demo.codimd.org/screenshot.png)
+## Documentation
+
+ * Official documentation: https://hackmd.io/c/codimd-documentation/
```

Comment:
Same as above, this will point to the wrong documentation.

Reply:
documentation link changed to https://github.com/codimd/server/tree/master/docs/